### DISCUSSION TITLE
cannot open webui graylog

### DISCUSSION THREAD

**Discussion by prauseclan** [RobertDWhite/WhiteMatterTech#16] [Q&A] [answered]
Hello Robert,
I want to deploy graylog on my unraid and I found your great instruction. Thanks for it.
But it seems I get some trouble to open the graylog url. I'm not so familiar with unraid and I'm a beginner. 
Maybe you got some ideas how I can solve it. For you information: my unraid is installed on a zimaboard.
The 3 containers are working.
When I click on the graylog container to open the webui it open my browser and put the following address: "http://192.168.1.115/192.168.1.115:9000". Also when I want to open http://192.168.1.115:9000 -> connection failed.
Find attached the docker compose file.
![version](https://github.com/RobertDWhite/WhiteMatterTech/assets/152654374/6f416c54-9794-47a6-8582-0f846932358c)
In the graylog logs I get the following information:
![graylogx](https://github.com/RobertDWhite/WhiteMatterTech/assets/152654374/c6bc008f-dbdf-4ea7-91a0-41f14536b75c)
![graylogx2](https://github.com/RobertDWhite/WhiteMatterTech/assets/152654374/ecb28342-76c4-4475-9199-e53080af741c)
![graylogx3](https://github.com/RobertDWhite/WhiteMatterTech/assets/152654374/95f18feb-bb9d-4d37-a054-5040dde684b8)
![graylogx4](https://github.com/RobertDWhite/WhiteMatterTech/assets/152654374/43aaf5e5-ee40-4649-a407-3ae942b9ddfc)
Regards Adam

**Comment by RobertDWhite** [comment, marked as answer]
@prauseclan thanks for the comment! 

The first thing I would want to try is upgrading mongodb to 5.0. If you replace the image from, 4.2 to 5.0 in the compose for mongodb and restart your container stack, do you get any different errors? Most of the issues I had with containerized Graylog in general were related to versioning. Try 5.0, and let me know what happens.

Happy New Year!
